Day2/BoxParser.py

```diff
@@ -0,0 +1,29 @@
+class BoxParser:
+    
+    raw_dimensions = None
+    parsed_dimensions = []
+    
+    def __init__(self, raw_dimensions):
+        self.raw_dimensions = raw_dimensions
+        self.parsed_dimensions = []
+    
+    def parse_single_box(self, box, delimiter="x"):
+        split_dimensions = box.split(delimiter)
+        int_dimensions = tuple(int(s) for s in split_dimensions)
+        return int_dimensions
+    
+    def is_tenth_line(self, floor, rob_wants_skipping=False):
+        if rob_wants_skipping:
+            return floor == 10
+        else:
+            return False
+    
+    def parse_all_boxes(self):
+        for i, box in enumerate(self.raw_dimensions, 1):
+            if self.is_tenth_line(i):
+                continue
+            parsed_box = self.parse_single_box(box)
```

Comment:
the box is a LIE 
es ezert megint megeroszakolod a valtozo neved "parsed" box
ha megfelelo a naming akkor box = self.parse_dimensions(dimensions)